RELEASE.md

```diff
@@ -0,0 +1,52 @@
+Release type: minor 
+
+Add `process_result` to views for Django, Flask and ASGI. They can be overriden
+to provide a custom response and also to process results and errors.
+
+It also removes `request` from Flask view's `get_root_value` and `get_context`
+since request in Flask is a global.
+
+Django example:
+
+```python
+# views.py
+from django.http import HttpRequest
+from strawberry.django.views import GraphQLView as BaseGraphQLView
+from strawberry.http import GraphQLHTTPResponse
+from strawberry.schema import ExecutionResult
+
+class GraphQLView(BaseGraphQLView):
+    def process_result(self, request: HttpRequest, result: ExecutionResult) -> GraphQLHTTPResponse:
+        return {"data": result.data, "errors": result.errors or []}
```

Comment:
Hmmmmmm I wonder if there is a way to do it that does not require writing a custom view anyway? Like `path('graphql', GraphQLView(process_result=custom_method_here))`? I still feel that it's more complicated than it needs to be?

Reply:
yeah, we could do that too, but it is not a huge difference, I'd keep this way :)